static/app/views/settings/organizationMembers/inviteBanner.spec.tsx

```diff
@@ -23,7 +24,7 @@ describe('inviteBanner', function () {
     MockApiClient.addMockResponse({
       url: '/organizations/org-slug/missing-members/',
       method: 'GET',
-      body: [],
+      body: [missingMembers],
```

Comment:
this is correct -- the API is returning the same thing aka `{integration: string, users: MissingMember[]}`. i'm returning an actual value here because the component now calls the API.

also before i was just passing in the `missingMembers` response into the component in testing and not actually using the mocked response